fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/OptExpressionDuplicator.java

```diff
@@ -244,6 +258,55 @@ public OptExpression visit(OptExpression optExpression, Void context) {
             return null;
         }
 
+        private HashDistributionSpec processHashDistributionSpec(
```

Comment:
add comment to describe what this function does

Reply:
done